### ISSUE TITLE
Migrate mock to unittest.mock

### ISSUE THREAD

**Issue by principis** [nvbn/thefuck#1262]
Hi, I'm updating thefuck for Fedora an stumbled across this issue. The mock package is deprecated in Fedora and we're trying to remove it.

Mock has been included in the standard library since python 3.3. Would you been open to migrate it to `unittest.mock`?

**Comment by scorphus** [comment]
Yes. There's a PR on that direction to which I should be soon giving attention: #1249

**Comment by principis** [comment]
Great, thanks! :slightly_smiling_face:
